mmv1/products/compute/RegionResizeRequest.yaml

```diff
@@ -0,0 +1,413 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: 'RegionResizeRequest'
+kind: 'compute#regionInstanceGroupManagerResizeRequest'
+min_version: beta
+description: |
+  Represents a Regional Managed Instance Group Resize Request
+
+  Resize Requests are the Managed Instance Group implementation of Dynamic Workload Scheduler Flex Start.
+
+  With Dynamic Workload Scheduler in Flex Start mode, you submit a GPU capacity request for your AI/ML jobs by indicating how many you need, a duration, and your preferred region. Dynamic Workload Scheduler intelligently persists the request; once the capacity becomes available, it automatically provisions your VMs enabling your workloads to run continuously for the entire duration of the capacity allocation.
+references:
+  guides:
+    # Link to quickstart in the API's Guides section. For example:
+    # 'Create and connect to a database': 'https://cloud.google.com/alloydb/docs/quickstart/create-and-connect'
+    'QUICKSTART_TITLE': 'https://cloud.google.com/compute/docs/instance-groups/create-resize-requests-mig'
```

Comment:
I think we're missing a "#" in this line. It also broke the formatting in the next lines

Reply:
Is it? In zonal, it's also don't have any "#" in the begining. And it's references to the docs about creating the resource.

Reply:
OK, if that's the same, then I take it back :)